### PR TITLE
Shaoqz/bigblue supp

### PR DESCRIPTION
**Pull request by shaoqx**
Add support for BigBlue
Also fixed a bug in the Gaussian Interface

### CODE REVIEW

File: enzy_htp/structure/chain.py
Comment (chrisjurich) [inline enzy_htp/structure/chain.py:1]: What lousy person put that there huh

File: setup.py
Comment (chrisjurich) [inline setup.py:1]: good catch!

**Review by chrisjurich — APPROVED** [review]
Looks good! Go big blue.

File: enzy_htp/structure/chain.py
Comment (shaoqx) [reply enzy_htp/structure/chain.py:1]: lol

**Pull request merged**